### PR TITLE
Add relay to docker build

### PR DESCRIPTION
**Pull request by pschork**
## Why are these changes needed?

Adds relay to docker builds

## Checks

- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
- [ ] I've checked the new test coverage and the coverage percentage didn't drop.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Integration tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -134,3 +134,7 @@ ENTRYPOINT ["nodeplugin"]
 FROM alpine:3.18 AS controller
 COPY --from=controller-builder /app/disperser/bin/controller /usr/local/bin
 ENTRYPOINT ["controller"]
+
+FROM alpine:3.18 AS relay
+COPY --from=common-builder /app/relay /usr/local/bin
+ENTRYPOINT ["relay"]
```
Comment (yujiezhu0) [inline Dockerfile:140]: We also need to add the build stage for relay, right?

**Comment by yujiezhu0** [comment]
I'm also editing docker bake-_- I guess either of us can rebase

File: Dockerfile (hunk 1 above)
Comment (pschork) [reply Dockerfile:140]: Thx

File: Dockerfile [hunk 2]
```diff
@@ -22,6 +22,7 @@ COPY contracts /app/contracts
 COPY indexer /app/indexer
 COPY encoding /app/encoding
 COPY relay /app/relay
+COPY inabox /app/inabox
```
Comment (dmanc) [inline Dockerfile:25]: Is this one needed?

Comment (pschork) [reply Dockerfile:25]: Yes. Docker Relay build fails without it.

**Review by yujiezhu0 — APPROVED** [review]
rebase needed

**Review by cody-littley — APPROVED** [review]

**Pull request merged**